internal/worker/rundynamic.go

```diff
@@ -63,6 +63,7 @@ func RunDynamicAnalysis(pkg *pkgecosystem.Pkg, sbOpts []sandbox.Option) (result.
 
 		results.StraceSummary[phase] = &result.StraceSummary
 		results.FileWrites[phase] = &result.FileWrites
+		results.FileWriteBuffers = append(results.FileWriteBuffers, result.FileWriteBuffers...)
```

Comment:
Should we split it up by phase to keep it in line with the other results?

Reply:
+1.

Reply:
Sounds good, so in this case we'd want the write files to be in separate folders in the output directory for each of the phases right?

Reply:
or is this mainly for us to store the data internally consistently?

Reply:
I think having them in separate folders is not such a bad idea. I mean, yes technically all the info about what each file means is in the JSON data, but it just seems like maintaining even a little bit of info about where each file comes from, by way of its file path, is a good thing to do. Even if it's just for some redundancy.

Reply:
For simplicity, I'd actually argue that we can just have a flat directory structure for file writes. All the necessary info can be gotten from the JSON data, and it's not clear if the redundancy here is necessary here.

Reply:
Fair enough, I'm not tied to either option

Reply:
Although we are no longer storing file write buffers we do need to store the paths to the files on disk to read from. I've stored those by phase.